### PR TITLE
Bug 1533611 Update openshift_logging_es_pvc_dynamic descripton

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1533611

### CODE REVIEW

File: install_config/aggregate_logging.adoc [hunk 1]
```diff
@@ -375,9 +375,14 @@ must be at least 512M. Possible suffixes are G,g,M,m.
...
 |`openshift_logging_es_pvc_dynamic`
-|If available for your
-cluster, set to `true` to have PVC claims annotated so that their
-backing storage is dynamically provisioned.
+|If available for your cluster, set to `true` to have PVC claims annotated so
+that their backing storage is dynamically provisioned. When set to `true` the
+storage class is ignored and not set to the PVC.
+
+|`openshift_logging_es_pvc_storage_class_name`
+|To use a non-default stoage class, set the variable with the storage class name.
```
Comment (jcantrill) [inline install_config/aggregate_logging.adoc]: ...non-default storage..

**Comment by bmcelvee** [comment]
@openshift/team-documentation PTAL :cherry_blossom:

File: install_config/aggregate_logging.adoc [hunk 2]
```diff
@@ -431,9 +431,14 @@ must be at least 512M. Possible suffixes are G,g,M,m.
 |A key/value map added to a PVC in order to select specific PVs.
 
 |`openshift_logging_es_pvc_dynamic`
-|If available for your
-cluster, set to `true` to have PVC claims annotated so that their
-backing storage is dynamically provisioned.
+|If available for your cluster, set to `true` to have PVC claims annotated so
+that their backing storage is dynamically provisioned. When set to `true` the
+storage class is ignored and not set to the PVC.
+
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: If *what* is available for your cluster? 

The "When set to `true`... line doesn't make sense to me (doesn't mean it is wrong). There is a BZ comment that uses *added* to the PVC. That seems to make more sense to me.

File: install_config/aggregate_logging.adoc [hunk 3]
```diff
@@ -431,9 +431,14 @@ must be at least 512M. Possible suffixes are G,g,M,m.
...
+|`openshift_logging_es_pvc_storage_class_name`
+|To use a non-default storage class, set the variable with the storage class name.
+For example, `openshift_logging_es_pvc_storage_class_name=glusterprovisioner` or
+`openshift_logging_es_pvc_storage_class_name=cephrbdprovisioner`.
```
Comment (mburke5678) [inline install_config/aggregate_logging.adoc]: Are these the only options?

**Comment by mburke5678** [comment]
@bmcelvee Couple of questions. Otherwise LGTM

**Pull request merged**